pytest.ini

```diff
@@ -65,3 +65,6 @@ filterwarnings =
     ignore:unclosed \<socket:ResourceWarning
     # Pending removal from sunpy 7.0
     ignore:The QueryResponse class is deprecated
+    # These should have been fixed by numpy 2.0 but possible its due to the
+    # other packages not building agasint it yet?
+    ignore:.*may indicate binary incompatibility.*
```

Comment:
This is wrong, as now they are valueerrors instead of warnings.

This will DO NOTHING